bcdaworker/main.go

```diff
@@ -204,9 +204,24 @@ func writeBBDataToFile(bb client.APIClient, acoID string, acoCMSID string, cclfB
 	totalBeneIDs := float64(len(cclfBeneficiaryIDs))
 	failThreshold := getFailureThreshold()
 	failed := false
+	suppressedList := models.GetSuppressedBlueButtonIDs()
 
 	for _, cclfBeneficiaryID := range cclfBeneficiaryIDs {
 		blueButtonID, err := beneBBID(cclfBeneficiaryID, bb)
+		foundSuppressed := false
+
+		for _, suppressedBBID := range suppressedList {
```

Comment:
What if we had `models.GetSuppressedBlueButtonIDs()` return a `map` with the BB IDs as keys? That should be faster than iterating. https://www.darkcoding.net/software/go-slice-search-vs-map-lookup/